daemon/cluster/executor/container/controller.go

```diff
@@ -211,6 +211,7 @@ func (r *controller) Start(ctx context.Context) error {
 				// Retry network creation again if we
 				// failed because some of the networks
 				// were not found.
+				log.G(ctx).WithError(lnErr).Warningf("Network not found when starting container %s, will create again", r.adapter.container.name())
```

Comment:
Not sure if this warrants a "warning" log (at _most_ a "recreating network .." _debug log_, but perhaps not even that?)

Reply:
This is a rare case, so we want to log it.

Reply:
Changed to debug log level.